srtcore/sync.h

```diff
@@ -241,8 +248,9 @@ class UniqueLock
 typedef UniqueLock CGuard;
 
 
-inline void enterCS(Mutex &m) { m.lock(); }
-inline void leaveCS(Mutex &m) { m.unlock(); }
+inline void enterCS(Mutex& m) { m.lock(); }
+inline bool maybeEnterCS(Mutex& m) { return m.try_lock(); }
+inline void leaveCS(Mutex& m) { m.unlock(); }
```

Comment:
Rename to `tryEnterCS`

Reply:
Later I was considering the same thing ;)